src/tests/cucumber/features/medium_tests.feature

```diff
@@ -1,11 +1,11 @@
 Feature: medium tests
 
-  @fast @medium @incomplete
+  @fast @medium @economy @incomplete
   Scenario: 035 Mixed Expansion - Smart grid model 2
     Given the study path is "medium-tests/035 Mixed Expansion - Smart grid model 2"
     When I run antares simulator
     Then the simulation takes less than 30 seconds
-    And the simulation succeeds
+    And the execution succeeds
```

Comment:
Let us put "the excution succeeds" first to prevent ambiguity when the first clause fail.